packages/rocketchat-mailer/package.js

```diff
@@ -8,32 +8,10 @@ Package.onUse(function(api) {
 	api.use([
 		'ecmascript',
 		'ddp-rate-limiter',
-		'kadira:flow-router',
-		'rocketchat:lib',
-		'rocketchat:authorization',
 	]);
+	Npm.depends({
```

Comment:
@ggazzo we are trying to keep the NPM dependencies on the root package.json file

@rodrigok can you help here?